tests/software/gubbins/test.yml

```diff
@@ -0,0 +1,23 @@
+- name: gubbins
+  command: nextflow run ./tests/software/gubbins -entry test_gubbins -c tests/config/nextflow.config
+  tags:
+    - gubbins
+  files:
+    - path: output/gubbins/test.filtered_polymorphic_sites.fasta
+      should_exist: true
```

Comment:
```suggestion
```

Reply:
If you just provide `path` then I think that by default checks for file existence. In any case, it would be great if we can have md5sum checks where possible here, and if they aren't possible e.g timestamps then you can also use `contains` for non-binary files to look for specific snippets of text. Means we can unit test the files a bit better.

See [docs here](https://pytest-workflow.readthedocs.io/en/stable/#test-options)

Reply:
An example on nf-core/modules here https://github.com/nf-core/modules/blob/989fb7d5b7db55d1fa305fad23877f95591f1e1a/tests/software/gatk4/variantfiltration/test.yml#L8-L9